tools/alignment_tools.py

```diff
@@ -0,0 +1,186 @@
+''' Tools for manipulating the alignments seralized file '''
+
+#TODO change format
+#TODO merge alignments
+
+import argparse
+import datetime
+import os
+import sys
+import json
+import pickle
+
+class AlignmentData(object):
+    ''' Class to hold the alignment data '''
+
+    def __init__(self, alignments_dir):
+        self.extensions = ['json', 'p', 'yaml', 'yml']
+        self.alignments_file = None
+        self.alignments_format = None
+        self.alignments = None
+
+        self.load_alignments(alignments_dir)
+
+    def alignments_file_exists(self, alignments_base, extension):
```

Comment:
I think it would be better to get the user to specify a path to a target alignments file as an input.
This would allow you to avoid all the checks for there being a single file.

Reply:
Yeah, that's probably correct. Amending